dependencies.tsv

```diff
@@ -52,7 +52,7 @@ github.com/juju/xml	git	eb759a627588d35166bc505fceb51b88500e291e	2015-04-13T13:1
 github.com/juju/zip	git	f6b1e93fa2e29a1d7d49b566b2b51efb060c982a	2016-02-05T10:52:21Z
 github.com/julienschmidt/httprouter	git	77a895ad01ebc98a4dc95d8355bc825ce80a56f6	2015-10-13T22:55:20Z
 github.com/lunixbochs/vtclean	git	4fbf7632a2c6d3fbdb9931439bdbbeded02cbe36	2016-01-25T03:51:06Z
-github.com/lxc/lxd	git	0d172e3080f768fd419f0c97f5246983797db243	2016-08-12T05:08:18Z
```

Comment:
Please, use `godeps github.com/juju/juju/... | grep lxc/lxd` to update that line, so it includes the correct date, not just the commit hash. Assuming your github.com/lxc/lxd work dir is at rev 48f.. it will produce the correct line to replace.